Demo/build.gradle

```diff
@@ -72,7 +72,7 @@ dependencies {
     implementation project(':Drop-In')
 
     debugImplementation 'com.facebook.stetho:stetho:1.5.0'
-    debugImplementation 'com.squareup.leakcanary:leakcanary-android:1.4'
+//    debugImplementation 'com.squareup.leakcanary:leakcanary-android:1.4'
```

Comment:
Should this be removed?

Reply:
Yeah leaning towards removing because these debug dependencies cause crashes sometimes and it isn't clear why. We also don't do anything with memory leak detection it might make sense to add this back after we GA the next major version of drop in.